docs/src/dataDisplay/dataCards/api/api.jsx

```diff
@@ -0,0 +1,29 @@
+import {
+    camelCase,
+} from 'lodash';
+import React from 'react';
+import ComponentApi from '../../../global/componentApi';
+import Main from '../../../global/main';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/dataDisplay/address/address';
```

Comment:
This is incorrect; it should be `dataCard`

Reply:
Fixed.

Reply:
Looks good.  Resolving.